libportability-gfx/src/conv.rs

```diff
@@ -450,12 +476,6 @@ pub fn map_stage_flags(stages: VkShaderStageFlags) -> pso::ShaderStageFlags {
     if stages & VkShaderStageFlagBits::VK_SHADER_STAGE_COMPUTE_BIT as u32 != 0 {
         flags |= pso::ShaderStageFlags::COMPUTE;
     }
-    if stages & VkShaderStageFlagBits::VK_SHADER_STAGE_ALL_GRAPHICS as u32 != 0 {
```

Comment:
Why don't we need these anymore?

Reply:
These are just combinations of the flags above.

Reply:
Ah ok